Godeps/_workspace/src/github.com/Clever/gearadmin/Makefile

```diff
@@ -0,0 +1,20 @@
+SHELL := /bin/bash
+PKG = github.com/Clever/gearadmin
+SUBPKGS =
```

Comment:
So, it seems strange to me that we're committing the gearmanadmin repo code... Shouldnt this be brought in as part of the build/deploy?